packages/grants/components/CommonComponents/SocialMedia.tsx

```diff
@@ -52,7 +52,7 @@ const SocialMedia: React.FC<SocialMediaProps> = ({ beneficiary }) => {
     <>
       <div className="flex space-x-6">
         <SocialMediaIcon url={beneficiary?.links?.website} platform={"website"} />
-        <SocialMediaIcon url={beneficiary?.links.contactEmail} platform={"email"} />
+        <SocialMediaIcon url={beneficiary?.links.contactEmail.data} platform={"email"} />
```

Comment:
Are we certain beneficiary?.links.contactEmail will always be defined? This will throw a typeerror if it's trying to access .contactEmail of undefined, so it might be good to add more "?" here.
```suggestion
        <SocialMediaIcon url={beneficiary?.links?.contactEmail?.data} platform={"email"} />
```